debug/botstats.py

```diff
@@ -0,0 +1,54 @@
+import sqlite3
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    import telegram
+    import telegram.ext
+
+conn = sqlite3.connect('/code/botstats.db', check_same_thread=False)
+cursor = conn.cursor()
+cursor.execute(f"CREATE TABLE IF NOT EXISTS `commands` ( "
+               "`command_name` VARCHAR(255) NOT NULL UNIQUE, "
+               "`command_count` INT unsigned NOT NULL DEFAULT '0', "
+               "PRIMARY KEY (`command_name`))")
+
+
+def print_botstats(update: 'telegram.Update', context: 'telegram.ext.CallbackContext') -> None:
+    """Get command usage stats"""
+    if not update.message:
+        return
+
+    text: str
+
+    # Query to get top 10 users by message count
+    formula: str = f"SELECT * FROM `commands` ORDER BY command_count DESC LIMIT 10"
+
+    if update.effective_chat.type != 'private':
```

Comment:
apparently `effective_chat` is an `Optional[chat]` type
```suggestion
    if update.effective_chat.type != 'private':  # type: ignore
```